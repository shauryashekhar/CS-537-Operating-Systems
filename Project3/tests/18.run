./mysh tests/18.in
