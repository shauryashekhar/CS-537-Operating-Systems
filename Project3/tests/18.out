wait 0
exit
